lib/events.js

```diff
@@ -479,23 +479,19 @@ EventEmitter.prototype.removeAllListeners =
     };
 
 EventEmitter.prototype.listeners = function listeners(type) {
-  var evlistener;
-  var ret;
-  var events = this._events;
+  const events = this._events;
 
-  if (events === undefined)
-    ret = [];
+  if (!events)
+    return [];
   else {
-    evlistener = events[type];
-    if (evlistener === undefined)
-      ret = [];
+    const evlistener = events[type];
+    if (!evlistener)
```

Comment:
Ditto.

Reply:
Same as above, total accident. Fixed now. Thanks for flagging!